Source/diablo.cpp

```diff
@@ -176,22 +176,22 @@ void initKeymapActions();
 
 [[noreturn]] static void print_help_and_exit()
 {
-	printInConsole("%s", _("Options:\n"));
-	printInConsole("    %-20s %-30s\n", "-h, --help", _("Print this message and exit"));
-	printInConsole("    %-20s %-30s\n", "--version", _("Print the version and exit"));
-	printInConsole("    %-20s %-30s\n", "--data-dir", _("Specify the folder of diabdat.mpq"));
-	printInConsole("    %-20s %-30s\n", "--save-dir", _("Specify the folder of save files"));
-	printInConsole("    %-20s %-30s\n", "--config-dir", _("Specify the location of diablo.ini"));
-	printInConsole("    %-20s %-30s\n", "--ttf-dir", _("Specify the location of the .ttf font"));
-	printInConsole("    %-20s %-30s\n", "--ttf-name", _("Specify the name of a custom .ttf font"));
-	printInConsole("    %-20s %-30s\n", "-n", _("Skip startup videos"));
-	printInConsole("    %-20s %-30s\n", "-f", _("Display frames per second"));
-	printInConsole("    %-20s %-30s\n", "-x", _("Run in windowed mode"));
-	printInConsole("    %-20s %-30s\n", "--verbose", _("Enable verbose logging"));
-	printInConsole("    %-20s %-30s\n", "--spawn", _("Force spawn mode even if diabdat.mpq is found"));
-	printInConsole("%s", _("\nHellfire options:\n"));
-	printInConsole("    %-20s %-30s\n", "--diablo", _("Force diablo mode even if hellfire.mpq is found"));
-	printInConsole("    %-20s %-30s\n", "--nestart", _("Use alternate nest palette"));
+	printInConsole("%s", _( /* TRANSLATORS: Commandline Option. No need to translate this.*/ "Options:\n"));
+	printInConsole("    %-20s %-30s\n", /* TRANSLATORS: Commandline Option. No need to translate this.*/  "-h, --help", _("Print this message and exit"));
+	printInConsole("    %-20s %-30s\n", /* TRANSLATORS: Commandline Option. No need to translate this.*/  "--version", _("Print the version and exit"));
+	printInConsole("    %-20s %-30s\n", /* TRANSLATORS: Commandline Option. No need to translate this.*/  "--data-dir", _("Specify the folder of diabdat.mpq"));
+	printInConsole("    %-20s %-30s\n", /* TRANSLATORS: Commandline Option. No need to translate this.*/  "--save-dir", _("Specify the folder of save files"));

```

Comment:
Should we not consider the CLI interface user facing?

Reply:
Welp, this might be a personal take but I think it's common to have English help messages in the CLI. I leave that up to y'alls judgment. :)

Reply:
I'm used to it being translated on Linux :)